docs/learn/learn-transaction-fees.md

```diff
@@ -24,23 +24,39 @@ designed the Polkadot fee system with the following objectives:
 
 Fees on the Polkadot Relay Chain are calculated based on three parameters:
 
-- A per-byte fee (also known as the "length fee").
-- A weight fee.
-- A tip (optional).
+- A Weight fee
+  - Base weight
+  - Call(s) weight
+- A Length fee
+- A Tip (optional).
 
-The length fee is the product of a constant per-byte fee and the size of the transaction in bytes.
-
-Weights are a fixed number designed to manage the time it takes to validate a block. Each
+Weights are a fixed number designed to manage the time to validate a block. Each
```

Comment:
The word "designing" seems a bit off as we are talking of numbers. Just refining the sentence 

```suggestion
Weights are a fixed set of numbers that are used in Substrate based chains to manage the time it takes to validate a block. 
```

Reply:
Thanks, would just edit to "Substrate-based"